pkg/controller/cronjob/utils_test.go

```diff
@@ -89,9 +89,10 @@ func TestGetJobFromTemplate2(t *testing.T) {
 	if len(job.ObjectMeta.Annotations) != 2 {
 		t.Errorf("Wrong number of annotations")
 	}
+	utcTimestamp := scheduledTime.UTC().Format(time.RFC3339)
```

Comment:
I think these thee strings are RFC3339 strings that represent the same instant:
- `2023-05-34T23:39:57-00:00`
- `2023-05-34T23:39:57+00:00`
- `2023-05-34T23:39:57Z`

However, we'd like to be consistent between our releases. Using a fixed string here would make it obvious to any reviewer when we're changing the outcome. That's why I recommend it.

Testing that `Format` returns the same result as another `Format` doesn't feel as helpful.